Library/Homebrew/formula_installer.rb

```diff
@@ -577,7 +577,11 @@ def install_dependencies(deps)
     elsif !deps.empty?
       oh1 "Installing dependencies for #{formula.full_name}: #{deps.map(&:first).map(&Formatter.method(:identifier)).join(", ")}",
         truncate: false
-      deps.each { |dep, options| install_dependency(dep, options) }
+      deps.each do |dep, options|
+        # Fix for patchelf
+        next if formula.full_name == dep.to_formula.full_name
+        install_dependency(dep, options)
+      end
```

Comment:
I propose to wait until 1.7.0 is merged, see if this still happens, and I'll open a PR for that. Are you OK with that?

Reply:
it doesn't hurt to ask... or does it? 😆

Reply:
It's just that I do not want to open a PR if this is fixed with whatever other magic has happened in the code since then. I want to avoid unnecessary discussions. I promise I'll open a PR as soon as possible :)